ADMVEDU95/ADMVEDU95/en.lproj/Localizable.strings

```diff
@@ -9,11 +9,11 @@
 "Start searching" = "Start searching";
 "song" = "song";
 "feature-movie" = "feature-movie";
-"Song" = "Song: %@";
+"_Song" = "Song: %@";
```

Comment:
why is underscore needed?

Reply:
Because for R.swift "song" and "Song" is equal

Reply:
in general keys should be structured kind of "X.Y.Z", e.g. "Search.Song.Title" and "Filters.Song.Title" or "Details.Song.Title" etc
Please structure all localisable keys